web/themes/new_weather_theme/templates/layout/page--local.html.twig

```diff
@@ -137,10 +137,10 @@
       {{ drupal_block("weathergov_daily_forecast") }}
 
       <div class="grid-row tablet:grid-col-12 margin-top-4">
-        <div class="tablet:grid-col-6 padding-right-2">
+        <div class="tablet:grid-col-6 tablet:padding-right-2">
           {{ drupal_block("weathergov_weather_story") }}
         </div>
-        <div class="tablet:grid-col-6 padding-left-2">
+        <div class="tablet:grid-col-6 tablet:padding-left-2">
```

Comment:
This is on me, but seeing it live, the 6 columns feels a little tight on a few of the tablet sizes. Could we adjust these to be `tablet:grid-col-8 desktop:grid-col-6`. The right padding can stay as is.

Reply:
👍 I'm curious how this will change when it's in the accordion but we'll see soon!

I don't think the paddings are necessary if we go to 8 columns, though. The padding was originally meant to create a gutter (is that the right term?) between the two-column weather story/WFO promo layout. But with the weather story being 8 columns, those two won't be in two columns.

Reply:
Yep you got it, that's gutter. Sometimes people call it the column gap, and it also corresponds with `grid-gap` in uswds. 

That makes sense, yeah I think WFO promo will be stacked below (at least for now ;) ). 

Looking at the latest commit I did just want to highlight that we do want to add `desktop:grid-col-6` to this one. On tablet I thought 8 columns worked better because the smaller screen made 6 columns a bit narrow, but on desktop those line lengths will get long with 8 columns.